<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fct_mod_1</class>
 <widget class="QDialog" name="fct_mod_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>566</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BD Théâtre</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabInsert">
      <attribute name="title">
       <string>Ajouter</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0" columnstretch="0,0,0">
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Entrez une date de représentation :</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Choisissez un spectacle</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Entrez une promotion pour cette représentation :</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QDoubleSpinBox" name="doubleSpinBox">
           <property name="minimum">
            <double>0.050000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.050000000000000</double>
           </property>
           <property name="value">
            <double>0.050000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QDateTimeEdit" name="dateTimeEdit">
           <property name="date">
            <date>
             <year>2019</year>
             <month>12</month>
             <day>2</day>
            </date>
           </property>
           <property name="minimumDateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2019</year>
             <month>12</month>
             <day>2</day>
            </datetime>
           </property>
           <property name="maximumDate">
            <date>
             <year>2020</year>
             <month>12</month>
             <day>20</day>
            </date>
           </property>
           <property name="minimumDate">
            <date>
             <year>2019</year>
             <month>12</month>
             <day>2</day>
            </date>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="comboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Valider</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSupp">
      <attribute name="title">
       <string>Supprimer</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Choisissez un spectacle</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox_2"/>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>Valider</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>300</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>dateRep</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>promoRep</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Supprimer cette représentation</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Fermer</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>fct_mod_1</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>fct_mod_1</receiver>
   <slot>refreshSupp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>718</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>fct_mod_1</receiver>
   <slot>refreshIns()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>815</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>fct_mod_1</receiver>
   <slot>SuppRep()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>648</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshIns()</slot>
  <slot>refreshSupp()</slot>
  <slot>SuppRep()</slot>
 </slots>
</ui>
